support-frontend/assets/pages/supporter-plus-landing/twoStepTest/firstStepLanding.tsx

```diff
@@ -114,6 +120,10 @@ export function SupporterPlusInitialLandingPage({
 			: paymentMethodsMarginRecurring}
 	`;
 
+	useEffect(() => {
+		dispatch(resetValidation());
+	}, []);
+
```

Comment:
note, this will catch users that press the browser back button on the 2nd checkout page instead of the change button. The only downside of having it here in a useEffect hook is that this will fire after the validation error message(s) have had a chance to render on the page as well, so you may get a brief flash of error messages before they go away. This seems fine for now until a more fundamental review of the checkout form validation is complete.